### DISCUSSION TITLE
Daily summaries combining all motion events into one video

### DISCUSSION THREAD

**Discussion by AskAlice** [blakeblackshear/frigate#7519] [Ideas]
In the zoneminder ecosystem there is this library https://github.com/ass-a2s/zmMagik
> this video is blended from 2 days worth of video. 
<img src="https://github.com/pliablepixels/zmMagik/raw/master/sample/1.gif" />
It appears this library also uses opencv, and a few other things. I imagine I would only need to port over the contents of these files.

[blend.py](https://github.com/ass-a2s/zmMagik/blob/master/zmMagik_helpers/blend.py)
[annotate.py](https://github.com/ass-a2s/zmMagik/blob/master/zmMagik_helpers/annotate.py)
[detect_yolo.py](https://github.com/ass-a2s/zmMagik/blob/master/zmMagik_helpers/detect_yolo.py)
[detect_background.py](https://github.com/ass-a2s/zmMagik/blob/master/zmMagik_helpers/detect_background.py)

Would this be an appropriate feature to include directly within frigate? Or would frigate prefer it if this functionality were provided as a separate repo that uses the API to query and process this information?

**Comment by NickM-27** [comment]
There is a pinned feature requests for this https://github.com/blakeblackshear/frigate/issues/54 

HOWEVER, the repo you linked has a license that is not compatible with Frigate's license, so the code could not be integrated into the frigate codebase.

**Reply by AskAlice** [reply]
I actually found that repo by searching for an implementation of a stackoverflow post seen here https://stackoverflow.com/questions/56429091/combining-features-of-all-videos-using-python
So it's fair to say this can be implemented without referencing that repo specifically

**Reply by NickM-27** [reply]
right, definitely, there wouldn't be a problem there as I understand it. Frigate also already has a lot of information in its DB and related things so likely wouldn't make sense to implement it in the same way.
